skeleton-navigation/src/about.ts

```diff
@@ -0,0 +1,3 @@
+export class About {
+    
```

Comment:
Maybe remove this file and make `about` an html-only custom element.

Reply:
done